elements/bulbs-reading-list/lib/reading-list-article.js

```diff
@@ -106,8 +106,39 @@ export default class ReadingListArticle {
     this.element.dataset.loadStatus = 'loading';
   }
 
+  setupTaboola () {
+    let taboola = window._taboola || [];
+    let taboolaContainer = document.createElement('div');
+    taboolaContainer.className = 'taboola-container';
+    taboolaContainer.id = 'taboola-below-article-text-links-' + (new Date()).getTime();
+    this.element.appendChild(taboolaContainer);
+    taboola.push({
+      mode: 'organic-text-links-c',
+      container: taboolaContainer.id,
+      placement: 'Below Article Text Links',
+      target_type: 'mix',
+    });
+    let taboolaItem = {
+      url: this.element.dataset.href,
+    };
+    if (this.element.dataset.isGraphic) {
+      taboolaItem.photo = 'auto';
+    } 
+    else if (this.element.dataset.isVideo) {
+      taboolaItem.video = 'auto';
+    } 
+    else {
+      taboolaItem.article = 'auto';
+    }
+    taboola.push(taboolaItem);
```

Comment:
I don't think we need this additional `taboolaItem` push?  I don't think that was in their original instructions